app/docs/enterprise/0.32-x/getting-started/adding-consumers.md

```diff
@@ -4,79 +4,78 @@ title: Adding Consumers
 
 # Adding Consumers
 
-In the last section, we learned how to add plugins to Kong Enterprise Edition
-(EE) - in this section
-we're going to learn how to add [Consumers][consumers] to Kong. Consumers are
-associated to individuals or applications using your API, and can be used
-for tracking, access management, and more.
+In the last section, we learned how to add plugins to Kong, in this section
```

Comment:
Kong entities like "plugins" should be capitalized per https://konghq.atlassian.net/wiki/spaces/PM/pages/2457689/Naming+and+Terminology+at+Kong+Inc.#NamingandTerminologyatKongInc.-Capitalization